asyncpg/protocol/record/recordobj.c

```diff
@@ -14,6 +14,21 @@
 # define _ApgObject_GC_IS_TRACKED PyObject_GC_IsTracked
 #endif
 
+#if PY_VERSION_HEX < 0x03070000
+static const char * _PyType_Name(PyTypeObject *type)
```

Comment:
Since `_PyType_Name` is not really public C API, I wouldn't mind if we used this version an all Pythons.

Reply:
That's a good idea. I took that approach in the latest revision.